pysoa/server/internal/types.py

```diff
@@ -1,76 +1,80 @@
-import six
+from __future__ import absolute_import, unicode_literals
+
+
+def get_switch(item):
+    if hasattr(item, '__int__'):
+        return item.__int__()
+    if hasattr(getattr(item, 'value', None), '__int__'):
+        return item.value.__int__()
+    return None
```

Comment:
I'd raise an error here rather than `None` - the `TypeError` from below, in fact.

Reply:
Changed.